content/rs/concepts/shard-placement-policy.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Shard Placement Policy
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/concepts/rebalancing-shard-placement/
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Shard Placement Policy
+
+The shard placement policies are:
+
+- dense - Place as many shards as possible on the smallest number of nodes to reduce the latency between the proxy and the database shards.
+- sparse - Spread the shards across as many nodes in the cluster as possible to spread the traffic across cluster nodes.
+
+When you create a cluster, the default shard placement policy on the cluster is **dense**.
+The default policy is assigned to all databases that you create on the cluster.
+
+### Dense Shard Placement Policy
+
+In the dense policy, shards are placed on the same node
+as long as the node has enough available memory for the memory allocated to the database.
+When the node is not able to host all of the shards, some shards are moved to another node to maintain optimal node health.
+
+For example, for a database with 2 master and 2 slave shards on a cluster with three nodes and a dense shard placement policy,
```

Comment:
s/2/two

Numbers 0-9 should be spelled out. See https://developers.google.com/style/numbers